src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,80 @@
+package duke;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.nio.file.Path;
+import java.util.ArrayList;
+
+import duke.task.Task;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.ObjectOutputStream;
+
+/** 
+ * Class which handles the underlying implementation of I/O from and to files.
+ *      1. Reading of tasks from a file and updating the current TaskList object;
+ *      2. Saving tasks from the current TaskList to the file in the filepath provided.
+ */
```

Comment:
Again, very neat and straightforward javadoc. However, try not to use pointers for your javadoc. 
Use: 
`* <li> Point one`
`* <li> Point two`
if necessary

Reply:
Didn't know about this before, and I've since modified the base code. Thank you!